File: src/main/resources/static/js/service/pdQuesService.js
//服务层
app.service('pdQuesService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('/pdQues/findAll');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('/pdQues/findPage?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('/pdQues/findOne?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('/pdQues/add',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('/pdQues/update',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('/pdQues/delete?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('/pdQues/search?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/group/yunxin/service/PdQuesService.java
package group.yunxin.service;
import java.util.List;
import group.yunxin.pojo.TbPdQues;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface PdQuesService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbPdQues> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbPdQues pdQues);
	
	
	/**
	 * 修改
	 */
	public void update(TbPdQues pdQues);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbPdQues findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbPdQues pdQues, int pageNum,int pageSize);
	
}
